global.json

```diff
@@ -1,5 +0,0 @@
-{
-  "sdk": {
-    "version": "2.2.108"

```

Comment:
Did you mean to delete this file?

Reply:
Yes. The build works without it and this way, we're always using the latest SDK installed on AppVeyor servers. This only works on AppVeyor servers.